.github/workflows/test.yml

```diff
@@ -6,6 +6,7 @@ jobs:
       matrix:
         ruby: [2.4, 2.5, 2.6, 2.7, jruby]
         platform: [ubuntu-latest, macos-latest, windows-latest]
+        rubyzip: [1.1.2, '']
```

Comment:
Usually, a test should be written for the fixed bug, so this is good. But I think it is not necessary for this case. I don't think gepub will use a new rubyzip API in the near future.

I propose to remove rubyzip variation test from the test matrix.

Reply:
> I don't think gepub will use a new rubyzip API in the near future.

Well, it happened unintentionally.

> I propose to remove rubyzip variation test from the test matrix.

Okay, updated PR.